cmd/gaia/cli_test/cli_test.go

```diff
@@ -49,13 +49,13 @@ func TestGaiaCLIMinimumFees(t *testing.T) {
 	txFees := fmt.Sprintf("--fees=%s", sdk.NewInt64Coin(denom, 23))
 	success, _, _ = f.TxSend(keyFoo, barAddr, sdk.NewInt64Coin(denom, 10), txFees)
 	require.True(f.T, success)
-	tests.WaitForNextNBlocksTM(1, f.Port)
+	tests.WaitForNextNBlocksTM(2, f.Port)
```

Comment:
Why wait for an additional block?

Reply:
Test was failing locally, seems to be fixed now, reverted change.